README.md

```diff
@@ -36,10 +36,10 @@ use ButterCMS\ButterCMS;
 $butterCms = new ButterCMS('<auth_token>');
 
 // Feeds - returns a SimpleXMLElement object
-$feed = $butterCms->getFeed('rss');
+$feed = $butterCms->fetchFeed('rss');
 
 // Posts
-$result = $butterCms->getPosts(['page' => 1]);
+$result = $butterCms->fetchPosts(['page' => 1]);
```

Comment:
@joshholat Can you add a simple example below here of iterating through posts and printing their titles? We find that basic examples can help new users not get hung up on getting up and running.

Also, note to @buttercms/core-team that onboarding/docs need to get updated to reflect this 2.0 update.